core/tasks.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import minicts
 import os
+import requests
```

Comment:
I would apply pep-8 import sorting (using isort) to this by putting third-party modules like requests and minicts into a block separate from the stdlib imports above.